fdbserver/storageserver.actor.cpp

```diff
@@ -299,6 +299,7 @@ struct StorageServer {
 	Arena lastArena;
 	double cpuUsage;
 	double diskUsage;
+	double currentRate = 1.0;
```

Comment:
Given the name currentRate, I would expect this to be an actual rate and not the probability of a request being accepted.

Reply:
https://en.wikipedia.org/wiki/Rate_function

Reply:
For what it's worth, I don't find it super straightforward from that wikipedia article what your response actually means. I took the initial comment as saying that a different name could more clearly describe what the value was since it doesn't seem to be a straightforward rate. Perhaps the name is defensible, though it's not clear to me why.